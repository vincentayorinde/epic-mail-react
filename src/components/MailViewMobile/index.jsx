import React from 'react';

const MailViewMobile = ({id, sender, receiver, title, messageBody, classes, back}) => (
    <div className={classes} id="readMessage">
      <div className="box-11-inbox">
      <i onClick={back} className="fas fa-chevron-left"></i>
      <i className="fas fa-trash-alt"></i>
      <i className="fas fa-reply"></i>
      <i className="fas fa-reply-all"></i>
      <i className="fas fa-forward"></i>
    </div>
      <div className="body-inbox">
      <h1>{title}</h1><br />
      <small>
        From: <strong> {sender}</strong> <br />
        To: <strong> {receiver}</strong>
      </small>
      <br />
      <br />
      {messageBody}
      </div>
    </div>
)

export default MailViewMobile;