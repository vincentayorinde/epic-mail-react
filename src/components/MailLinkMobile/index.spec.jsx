
import React from 'react';
import { shallow } from 'enzyme';
import MailViewMobile from './index';

describe('MailViewMobile Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<MailViewMobile />);
    expect(component.find('.record').length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
