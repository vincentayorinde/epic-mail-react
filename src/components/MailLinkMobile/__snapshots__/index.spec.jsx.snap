// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailViewMobile Component Should render without errors 1`] = `
<div
  className="view record"
>
  <div
    className="view pic"
  >
    <img
      alt=""
    />
  </div>
  <div
    className="view record-inner"
  >
    <span
      className="view"
    />
    <br />
    <span
      className="view"
    >
       
      <h3>
        <i />
         
      </h3>
       
    </span>
    <h4
      className="view"
    />
  </div>
</div>
`;
